systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -322,4 +324,13 @@ public static String changeDeploymentNamespace(File deploymentFile, String names
             throw new RuntimeException(e);
         }
     }
+
+    public static void waitForResourceReadiness(String resourceType, String resourceName) {
+        LOGGER.info("Waiting for " + resourceType + "/" + resourceName + " readiness");
+
+        TestUtils.waitFor("resource " + resourceType + "/" + resourceName + " readiness",
+                Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_CMD_CLIENT_TIMEOUT,
+            () -> ResourceManager.cmdKubeClient().getResourceReadiness(resourceType, resourceName));
+        LOGGER.info("Resource " + resourceType + "/" + resourceName + " is ready");
+    }
```

Comment:
Wouldn't be better to add this to `ResourceManager`? Both options - have it in `StUtils` or `ResourceManager` - makes sense to me. So it's up to you

Reply:
Make sense to move it to ResourceManager